api/Console.json

```diff
@@ -306,15 +306,10 @@
             "ie": {
               "version_added": "10"
             },
-            "nodejs": [
-              {
-                "version_added": "8.10.0",
-                "notes": "Alias for <code>console.log</code>"
-              },
-              {
-                "version_added": "8.0.0"
-              }
-            ],
+            "nodejs": {
+              "version_added": "8.0.0",
+              "notes": "Since NodeJS 8.10.0, this method is an alias for <code>console.log</code>."
```

Comment:
```suggestion
              "notes": "Since Node.js 8.10.0, this method is an alias for <code>console.log</code>."
```
```suggestion
              "notes": "Since NodeJS 8.10.0, this method is an alias for <code>console.log</code>."
```